src/config/api.js

```diff
@@ -1,6 +1,7 @@
 import { create } from 'apisauce';
+import { camelCase, snakeCase, mapKeys } from 'lodash';
```

Comment:
I recommend you the use of that library https://www.npmjs.com/package/cerealizr instead of lodash 

```JS 
import { CamelcaseSerializer, SnakecaseSerializer } from 'cerealizr';

export const defaultCamelcase = new CamelcaseSerializer();

export const defaultSnakecase = new SnakecaseSerializer();


/* use at the code */
api.addResponseTransform(response => {
    if (response.data) {
      response.data = deserializer.serialize(response.data);
    }
  });

api.addRequestTransform(request => {
    if (request.data) {
      request.data = serializer.serialize(request.data);
    }
  });
```

Reply:
Ok, I used this approach. thanks for the feedback.